common/src/main/java/com/bakdata/dedupe/clustering/RefineCluster.java

```diff
@@ -139,7 +139,7 @@ private static double scoreClustering(final byte[] partitions, final double[][]
                 .mapToObj(i -> {
                     // reverse of Gaussian
                     int leftIndex = (int) (Math.sqrt(i + 0.25) - 0.5);
```

Comment:
First error, it should be: int leftIndex = (int) (Math.sqrt(2 * i + 0.25) - 0.5);

Reply:
Explanation: Gaussian formula is i = n * (n + 1) / 2, with n is leftIndex
To solve: 0 = n^2 + n - 2 * i